src/sentry/models/releasefile.py

```diff
@@ -101,6 +101,11 @@ def normalize(cls, url):
             urls.append("~" + urlunsplit(uri_relative_without_query))
         return urls
 
+    @classmethod
+    def public_objects(cls):
```

Comment:
nit: I think this is a task for a [Manager](https://docs.djangoproject.com/en/3.2/topics/db/managers/), then it's easier to chain in existing query with filters.
- in sentry e.g. - https://github.com/getsentry/sentry/blob/master/src/sentry/models/project.py#L53

Reply:
Thanks for the hint, I rewrote it now according to this: https://docs.djangoproject.com/en/1.11/topics/db/managers/#modifying-a-manager-s-initial-queryset